### DISCUSSION TITLE
Simple Monorepos with Rye, using the Polylith Architecture

### DISCUSSION THREAD

**Discussion by DavidVujic** [astral-sh/rye#571] [Show and tell]
Hi 👋 

I learned about Rye a few days ago and have done some reading up on what it does and how to use it. Very cool tool! ⭐ 

When learned some of the basics, I wanted to see if I could use Rye with Polylith (I am the maintainer of the [Polylith tool ](https://github.com/DavidVujic/python-polylith) for Python), and it turns out it works really well! Thanks to the option of using Hatch or PDM as the build backend, that the Polylith tool already support.

I got inspired an recorded an improvised video (there's a repo available too) about the Developer Experience with Hatch, Rye and Polylith and hope it is interesting and useful for you guys too:

https://youtu.be/BXPQBXuiRwM?si=Mwz4QZibvYmDvwVP

**Comment by ihatemodels** [comment]
Thank you !
